### PR TITLE
Remove blueocean dependency

### PR DESCRIPTION
**Pull request by agentgonzo**
Remove the dependencies on blueocean.

@reviewbybees

### CODE REVIEW

**Comment by jtnord** [comment]
THis is likely to blow up when used from BO as a  dependency?  is there plans to change all the pacakges?

File: src/main/java/io/jenkins/blueocean/commons/ErrorMessage.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ *     "errors" : [ {
+ *          "message" : "Invalid password",
+ *          "code" : "INVALID",
+ *          "field" : "password"
+ *     } ]
+ * }
+ *
+ * @author Vivek Pandey
+ **/
+public class ErrorMessage {
```
[the remark this answers is not in the document]
Comment (agentgonzo) [reply src/main/java/io/jenkins/blueocean/commons/ErrorMessage.java]: renamed in #3

File: src/main/java/io/jenkins/blueocean/commons/ServiceException.java [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+        }
+
+        public PreconditionRequired(ErrorMessage errorMessage) {
+            super(PRECONDITION_REQUIRED, errorMessage, null);
+        }
+        public PreconditionRequired(ErrorMessage errorMessage, Throwable throwable ) {
+            super(PRECONDITION_REQUIRED, errorMessage, throwable);
+        }
+    }
+    public static final int BAD_REQUEST = 400;
```
[the remark this answers is not in the document]
Comment (agentgonzo) [reply src/main/java/io/jenkins/blueocean/commons/ServiceException.java]: This was just copied over from blueocean repo

File: src/main/java/io/jenkins/blueocean/commons/ServiceException.java [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+/**
+ * This exception class to be used by all service methods.
+ *
+ * An error is represented by a code, in case of HTTP transport it could be http specific error code.
+ *
+ * Error message is represented by {@link ErrorMessage}
+ *
+ * @author Vivek Pandey
+ */
+public class ServiceException extends RuntimeException implements HttpResponse {
```
[the remark this answers is not in the document]
Comment (agentgonzo) [reply src/main/java/io/jenkins/blueocean/commons/ServiceException.java]: I think we can probably remove the commons package and change the exceptions as we use so little of this. We could start using some of the org.aacegisecuryty exceptions (that hudson extends anyway)

File: src/main/java/io/jenkins/blueocean/commons/JsonConverter.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.StringReader;
+import java.text.SimpleDateFormat;
+
+/**
+ * @author Vivek Pandey
+ **/
+public class JsonConverter{
```
[the remark this answers is not in the document]
Comment (agentgonzo) [reply src/main/java/io/jenkins/blueocean/commons/JsonConverter.java]: Will look at this is a subsequent PR

**Pull request merged**